### PR TITLE
👨🏼‍⚖️ Council card

### LINKED ISSUE #1381: Candidate card component
@[figma](https://www.figma.com/file/GlgN8uBRtvtMJtiOsdtDF7/Pioneer-Design?node-id=1883%3A641)

Implement the council candidate component

- [x] Member info (Avatar, handle + flags)
- [x] Card image (as empty for now)
- [x] Candidacy title
- [x] Candidacy rationale (handle cutting out bigger text)
- [x] Display stake
- [x] Candidate history:
  - [x] Newcomer
  - [x] Past wons/loses
- [x] Actions
  - [x] Vote
  - [x] Vote again (?)
  - [x] Withdraw candiacy (for matching member)
- [x] The card should work as well in the "preview mode" (link to the issue)

### PR DESCRIPTION
**Pull request by ClumsyVlad**
Closes #1381 

If actions 'onClick' for 'Vote', 'Vote again' and 'Withdraw Candidacy' aren't necessary for now then please check tasks in issue because I've only implemented this buttons without actions 🤔

### CODE REVIEW

**Comment by jodator** [comment]
> If actions 'onClick' for 'Vote', 'Vote again' and 'Withdraw Candidacy' aren't necessary for now then please check tasks in issue because I've only implemented this buttons without actions

That's good. I think that we're going to implement them all as most of them look essential.

**Review by jodator — COMMENTED** [review]
The component is generally working really nice :)

However, since we don't need it right now, we could refine it a bit.

**Comment by jodator** [comment]
And one from Natalia :) The handle should be aligned to member's avatar (there was a bug on [Figma design](https://www.figma.com/file/GlgN8uBRtvtMJtiOsdtDF7/Pioneer-Design?node-id=1883%3A641)).

**Comment by jodator** [comment]
And one from Natalia :) The handle should be aligned to member's avatar (there was a bug on [Figma design](https://www.figma.com/file/GlgN8uBRtvtMJtiOsdtDF7/Pioneer-Design?node-id=1883%3A641)).

File: packages/ui/src/forum/components/Council/CandidateCard.tsx [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+  title: string
+  infolist?: string[]
+  stake?: BN
+  wons?: number
+  losts?: number
+}
+
+export interface CandidateCardProps {
+  candidate: Candidate
+}
```
[the remark this answers is not in the document]
Comment (ClumsyVlad) [reply packages/ui/src/forum/components/Council/CandidateCard.tsx]: What should I do with withdrawable prop for now?

[the remark this answers is not in the document]
Comment (jodator) [reply packages/ui/src/forum/components/Council/CandidateCard.tsx]: Ah, we can work on that later. For now we can leave it as is.

**Review by jodator — APPROVED** [review]
Thanks!

**Pull request merged**